src/hooks/words/use-words-with-semesters.hook.ts

```diff
@@ -15,7 +15,7 @@ export const useWordsWithSemesters = () => {
   const [, onGetWords] = useWords()
   const onGetSemesters = useSemesters()
 
-  const onGetWordsWithSemesters = useCallback(async () => {
+  const onWordWithSemesters = useCallback(async () => {
```

Comment:
onGetWordWithSemesters

Reply:
Handled in [5ddf33e](https://github.com/ajktown/wordnote/pull/158/commits/5ddf33e05136c5e01ce02ca05840754462e6215c)